convert2rhel/unit_tests/pkgmanager/handlers/yum/yum_test.py

```diff
@@ -383,19 +383,21 @@ def test_resolve_yum_problematic_dependencies(
     monkeypatch,
     caplog,
 ):
+    monkeypatch.setattr(pkgmanager.handlers.yum.backup, "backup_control", mock.Mock())
+    monkeypatch.setattr(pkgmanager.handlers.yum, "RestorablePackage", mock.Mock())
     monkeypatch.setattr(pkgmanager.handlers.yum, "remove_pkgs", RemovePkgsMocked())
     pkgmanager.handlers.yum._resolve_yum_problematic_dependencies(output)
 
     if expected_remove_pkgs:
         assert pkgmanager.handlers.yum.remove_pkgs.called
-        pkgmanager.handlers.yum.remove_pkgs.assert_called_with(
-            pkgs_to_remove=expected_remove_pkgs,
-            backup=True,
-            critical=True,
-            reposdir=utils.BACKUP_DIR,
+        assert pkgmanager.handlers.yum.RestorablePackage.called
+        pkgmanager.handlers.yum.RestorablePackage.assert_called_with(
+            pkgs=expected_remove_pkgs,
+            reposdir=backup.BACKUP_DIR,
```

Comment:
Maybe I would mock the backup dir to use tmpdir. It can help with avoiding troubles in unit tests

Reply:
We are mocking the entire class here, it should be fine without mocking the backup dir